### PR TITLE
Meson bump

### PR DESCRIPTION
**Pull request by AndersonTorres**
###### Motivation for this change

Get rid of old Meson

https://github.com/NixOS/nixpkgs/pull/144779

As I have said before, the plan is:

- Insert new Meson in parallel with old one, the older being a safe default (done)
- Try to force new Meson as default
- Look at the ofborg logs in order to repair those packages that do not accept new Meson yet
  - Updates should be preferrable

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/libraries/wlroots/0.15.nix [hunk 1]
```diff
@@ -1,7 +1,28 @@
...
+, pkg-config
+, seatd
+, vulkan-loader
+, wayland
+, wayland-protocols
+, wayland-scanner
+, xcbutilerrors
+, xcbutilimage
+, xcbutilrenderutil
+, xcbutilwm
```
Comment (primeos) [inline pkgs/development/libraries/wlroots/0.15.nix:25]: Not a blocker but I hate those refactorings in unrelated commits (it's a bad commit practice). We should either reformat all of Nixpkgs at once or packages individually. A commit like https://github.com/NixOS/nixpkgs/pull/153082/commits/3d74e5725990dee4b934a8a1be8f8aa615c5da9e ("wlroots 0.15: change meson attribute - Meson 0.60 is the default now") shouldn't contain such unrelated changes.

Comment (AndersonTorres) [reply pkgs/development/libraries/wlroots/0.15.nix:25]: Understandable.

I prefer to do such code refactorings among other things because they did not change the semantics of the code.
